spec/requests/api/v2/green_lanes/categorisation_controller_spec.rb

```diff
@@ -1,15 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe Api::V2::GreenLanes::CategorisationsController do
+  before do
```

Comment:
We ought to not mock the env if we can help it

Reply:
Can we maybe encapsulate the env var in the application config module TradeTariffBackend and mock that?

Reply:
Done this with moving to TradeTariffBackend and mocking it instead of ENV